src/scope.js

```diff
@@ -15,4 +15,5 @@
 
 typedOM = {};
 baseClasses = {};
-
+if (!TYPED_OM_TESTING)
+	var typedOMTesting = null;
```

Comment:
don't you need to set typedOMTesting to an empty object (or to window) if TYPED_OM_TESTING is true?